src/components/Projects/index.tsx

```diff
@@ -267,8 +278,8 @@ export default function Projects() {
             }}
           >
             {concatenatedPages?.length}{' '}
```

Comment:
**suggstion**: lingui has a nicer way of handling plurals so that we can avoid splitting sentences.

I don't have time to give a concrete suggestion, but take a look at https://lingui.js.org/guides/plurals.html

Reply:
This is great, but causing problems I couldn't figure out. `plurals` isn't importing for lingui/core `i18n`. I will get this done in another PR